src/main/scala/cromwell/engine/backend/jes/JesAttributes.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend.jes
+
+import java.net.URL
+
+import com.typesafe.config.{Config, ConfigFactory}
+import cromwell.util.ConfigUtil._
+
+case class JesAttributes(applicationName: String,
+                         project: String,
+                         executionBucket: String,
+                         endpointUrl: URL,
+                         authMode: GcsAuthMode,
+                         docker: Option[DockerAuthInformation])
+object JesAttributes {
+
+  def apply(): JesAttributes = {
+    val jesConf = ConfigFactory.load.getConfig("backend").getConfig("jes")
+    val refConf: Config = ConfigFactory.parseResources("jes.conf")
+
+    jesConf.checkValidWrapped(refConf, "Jes")
+    jesConf.warnNotRecognized(refConf, "Jes", "dockerAccount", "dockerToken")
+
+    val applicationName = jesConf.getString("applicationName")
+    val project = jesConf.getString("project")
+    val executionBucket = jesConf.getString("baseExecutionBucket")
+    val endpointUrl = jesConf.getURL("endpointUrl")
+    val authMode = GcsAuthMode.fromString(jesConf.getString("authenticationMode"))
+    val docker = for {
+      account <- jesConf.getStringOption("dockerAccount")
+      token <- jesConf.getStringOption("dockerToken")
+    } yield DockerAuthInformation(account, token)
+
```

Comment:
Would it make sense to use Validations here? JES config now has oodles of options all of which could throw and such. It'd be nice to get a feel for everything which went wrong at once. That also might simplify htat deeply nested catch construct above since we'd have better control over the throwing